doc/integration.md

```diff
@@ -126,6 +129,94 @@ Note the following:
 ###When should you show the registration view?
 While we've seen how to utilize the registration view, we never talked about **when** you should display it. While it is ultimately up to you to decide, we have a fairly straightforward recommendation - you should only display the registration view when you do not have a user token stored. Assuming your user has gone through the registration process successfully and received a token there is no need to have the user register again, as tokens do not expire (though they can be revoked).
 
+##Performing purchases
+The aim of this SDK is to allow users to make purchases using your application, backed by Klarna as a payment method. However, the SDK does not offer any direct methods for performing purchases as this will expose your application's private Klarna credentials. Instead, applications using the SDK are expected to work in concert with an application backend, which will perform the actual purchase requests.
+
+In this section, we will see how to communicate with such a backend and for that purpose we supply a sample backend that you can find [here](https://github.com/klarna/sample-ondemand-backend). Reading the sample backend's documentation will allow you to fully grasp how an application using this SDK is expected to perform purchases, and you are encouraged to take a look if things become too unclear.
+
+###Signing requests
+While you can, and almost certainly will, communicate with your application's backend in a way that is different from the very simplistic approach we present here, one thing you will always have to do is sign your purchase requests. This will significantly increase your user's security while buying and the SDK makes this task incredibly easy.
+
+Let us say a user wants to make a purchase for a total of 40.50 Euros. All that's necessary to generate the relevant signature is to import the origin proof class:
+
+```java
+import com.klarna.ondemand.OriginProof;
+```
+
+which will allow you to construct an object as seen below:
+
+```java
+OriginProof originProof = new OriginProof(4050, "EUR", getUserToken(), getApplicationContext());
+```
+
+Assume `getUserToken()` returns the user's token as received during registration. Note that the constructor expects the purchase amount to be supplied in cents. You can find the constructor's full documentation [here](http://klarna.github.io/klarna-on-demand-android/com/klarna/ondemand/OriginProof.html#OriginProof-int-java.lang.String-java.lang.String-android.content.Context-).
```

Comment:
As we discussed, I am pretty sure most people are not familiar with the term "minor currency unit" and will only be confused by it.